src/ui/public/chrome/directives/app_switcher/app_switcher.less

```diff
@@ -79,13 +79,20 @@ body { overflow-x: hidden; }
     height: @app-icon-height;
     line-height:  @app-line-height;
 
-
     > a {
       display: block;
       height: 100%;
       color: #ebf7fa;
     }
 
+    &.is-app-switcher-app-link-disabled {
+      opacity: 0.5;
+
+      a {
```

Comment:
Instead of using an element selector, it would be better to add a class to the anchor tag in app_switcher.html, and refer to that class here. This is more maintainable if, for whatever reason, we decide to add another link inside of the app link component (I've seen this happen at a previous job, believe it or not).